pydicomtest/print_elements.go

```diff
@@ -57,7 +57,11 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	data, err := dicom.ReadDataSetInBytes(bytes, dicom.ReadOptions{})
+	p, err := dicom.NewParserFromBytes(bytes, nil)
```

Comment:
for folks who just want to parse a file and not do any intermediate header checking we can likely make a helper function to collapse this